odoo/tools/populate.py

```diff
@@ -0,0 +1,72 @@
+import random
+from odoo.tools import pycompat
+
+def _randomizer(seed):
+    r = random.Random()
+    r.seed(seed, version=2)
+    return r
```

Comment:
This looks trivially the same as
```suggestion
    return random.Random(seed)
```

Reply:
Except that version is not explicitly given, if I remember well Anthony asked me to ensure that the random version stays the same. Maybe not useful now, I can remove it if you really want.